sdk/core/core-http/tsconfig.json

```diff
@@ -3,7 +3,6 @@
   "compilerOptions": {
     "module": "commonjs",
     "outDir": "dist",
-    "target": "es5",
```

Comment:
bump the major for this and core-lro?

Reply:
and reset the min version for packages that were consuming old es5 core-http

Reply:
done.